frontend/packages/console-dynamic-plugin-sdk/CHANGELOG-core.md

```diff
@@ -1,7 +1,18 @@
 # Changelog for `@openshift-console/dynamic-plugin-sdk`
 
-Refer to [Console dynamic plugins README](./README.md) for OpenShift Console version vs SDK package
-version and PatternFly version compatibility.
+Console plugin SDK packages follow a semver scheme where the major and minor version number indicates
+the supported OCP Console version, and the patch version number indicates the release of that particular
```

Comment:
Wondering if we should make a comment about forward compatibility, or at least say

```suggestion
the earliest supported OCP Console version, and the patch version number indicates the release of that particular
```

Reply:
Good idea :+1: I think this is an important clarification.